bin/build_structure_file.rb

```diff
@@ -37,23 +39,44 @@ def get_iban_structures(iban_structures_file, iban_registry_file)
       account_number_position: country.account_number_position.to_i,
       account_number_length: country.account_number_length.to_i,
       total_length: country.total_length.to_i,
-      national_id_length: country.national_id_length.to_i
+      national_id_length: country.national_id_length.to_i,
     }.merge(bban_formats[country.country_code])
   end
 end
-
+# rubocop:enable Metrics/AbcSize
+
+# 0 Data element
+# 1 Name of country
+# 2 IBAN prefix country code (ISO 3166)
+# 3 Country code includes other countries/territories
+# 4 SEPA country
+# 5 SEPA country also includes
+# 6 Domestic account number example
+# 7 BBAN
+# 8 BBAN structure
+# 9 BBAN length
+# 10 Bank identifier position within the BBAN
+# 11 Bank identifier pattern
+# 12 Branch identifier position within the BBAN
+# 13 Branch identifier pattern
+# 14 Bank identifier example
+# 15 Branch identifier example
+# 16 BBAN example
+# 17 IBAN
+# 18 IBAN structure
+# 19 IBAN length
+# 20 Effective date
+# 21 IBAN electronic format example
 def get_bban_formats(iban_registry_file)
   iban_registry_file.each_with_object({}) do |line, hash|
-    bban_structure = line['BBAN structure '].strip
+    bban_structure = line[8].strip
+
+    bank_code_structure = line[11].strip
+    branch_code_structure = line[13].strip || line[12].strip
```

Comment:
> The iban registry txt file has been transposed 🤷

Is this why we now have the hardcoded numbers and the key above?

BOLD decision.

Is there value in not only keeping the mapping but for the numbers we do care about give them less magic? For example:

 ```
line[8].strip

=>

BBAN_STRUCTURE = 8
line[BBAN_STRUCTURE].strip
```

Reply:
>Is this why we now have the hardcoded numbers and the key above?
Yeah it's now column based rather than row which is fun.

>Is there value in not only keeping the mapping but for the numbers we do care about give them less magic?
Yeah this is a good plan, I'll add this

Reply:
Added 🎉

Reply:
> Yeah it's now column based rather than row which is fun.

Making it a lil more exciting in lockdown